### PR TITLE
initializer list for Coord & FloatCoord

### PR DESCRIPTION
**Pull request by V3RGANz**
initializer list implementation to avoid bulky use as functions / methods parameters

### CODE REVIEW

File: src/libgeodecomp/geometry/coord.h [hunk 1]
```diff
@@ -117,6 +117,11 @@ class Coord<1>
         c[0] = int(other[0]);
     }
 
+    inline explicit Coord(std::initializer_list<int> list)
```
Comment (gentryx) [inline src/libgeodecomp/geometry/coord.h]: This code is specific to C++11. Could you wrap it in preprocessor guards ("ifdef LIBGEODECOMP_WITH_CPP14")? (In LibGeoDecomp we're basically distinguishing between a C++98 compatibility mode and C++14, that's why there is no LIBGEODECOMP_WITH_CPP11).

Comment (gentryx) [reply src/libgeodecomp/geometry/coord.h]: Same below.

File: src/libgeodecomp/geometry/coord.h [hunk 2]
```diff
@@ -624,6 +642,19 @@ class Coord<3>
         c[2] = int(other[2]);
     }
 
+    inline explicit Coord(std::initializer_list<int> list)
+    {
+        int i = 0;
+        for (int coord : list){
```
Comment (gentryx) [inline src/libgeodecomp/geometry/coord.h]: Please add a space between the closing parenthesis and the opening curly brace.

File: src/libgeodecomp/geometry/coord.h [hunk 3]
```diff
@@ -624,6 +642,19 @@ class Coord<3>
...
+            c[i] = coord;
+            if (i > 2)
```
Comment (gentryx) [inline src/libgeodecomp/geometry/coord.h]: Please add curly braces around the statement after the conditional. LGD is using 1TBS as coding style.

**Comment by V3RGANz** [comment]
remove "explicit's" + fixed initializer_list's parameter for FloatCoord<3>

**Comment by gentryx** [comment]
LGTM, thanks!

**Pull request merged**